web/screenshot/app.screenshot-spec.ts

```diff
@@ -0,0 +1,11 @@
+import { AppPage } from './app.po';
```

Comment:
This import doesn't seem to be used.

Reply:
I tried running the tests without that import statement and there was a `DevToolsActivePort file doesn't exist` error. The import statement imports in the app.po.ts file which helps navigate to the browser. While I was looking at the app.po.ts file, I did realize I had an unused command of getting the text title, which I now removed